src/Germany/Mews.Fiscalizations.Germany/Model/Result/ResponseResult.cs

```diff
@@ -15,10 +17,7 @@ internal ResponseResult(TResult successResult = null, ErrorResult errorResult =
 
         public bool IsSuccess
         {
-            get
-            {
-                return SuccessResult != null;
-            }
+            get { return ErrorResult.IsNull(); }
```

Comment:
There's a story behind this change. Where it can happen that both results are null.

Reply:
When we get nothing in the response (example: Login/Logout)